tox.ini

```diff
@@ -3,7 +3,7 @@ addopts = --reuse-db
 DJANGO_SETTINGS_MODULE = tests.settings
 
 [tox]
-envlist = py27-django16, flake8
+envlist = py27-django16
```

Comment:
For now, yes. I don't really want to be seeing all the failures on those locally when I run the tests. Once I'm done with the tests and start working on the upgrades and changes, then those will be enabled.